datumaro/cli/commands/generate.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (C) 2020-2021 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import argparse
+import logging as log
+import os
+import os.path as osp
+from shutil import rmtree
+
+from datumaro.cli.util.errors import CliException
+from datumaro.plugins.synthetic_data import ImageGenerator
+
+from ..util import MultilineFormatter
+
+
+def build_parser(parser_ctor=argparse.ArgumentParser):
+    parser = parser_ctor(
+        help="Generate synthetic dataset",
+        description="""
+        Creates a synthetic dataset with elements of the specified shape and
+        saves it in the provided directory.|n
+        To create 3-channel images, you should provide height and width for them.|n
+        |n
+        Examples:|n
+        - Generate 300 3-channel synthetic images with H=224, W=256 and store to data_dir:|n
+        |s|s%(prog)s -o data_dir -k 300 --shape 224 256|n
+        """,
+        formatter_class=MultilineFormatter,
+    )
+
+    parser.add_argument(
+        "-t", "--type", required=True, choices=["image"], help="Specify type of data to generate"
```

Comment:
Can we provide default while we have only 1 option?

Reply:
done